### ISSUE TITLE
Automatic slot booking

### ISSUE THREAD

**Issue by MichielTG** [uipko/toplogger_notifier#2]
Is there a way to add automatic reservations via the API? I found the URL https://api.toplogger.nu/v1/reservations/, however, it needs login permission {"error":"You need to sign in or sign up before continuing."}.

**Comment by uipko** [comment]
Login is not an issue, login is needed for all API calls. My script is using the login before it is searching for empty slots.

But to make a reservation you need to fill in a recaptcha and these are designed to stop scripts/bots.

It would be a very nice feature. If you find a way to automate the reservation, please create a pull request.

**Comment by MichielTG** [comment]
I will let you know. Btw. I think login is not required if you want to view the open spots. For example, see:
https://api.toplogger.nu/v1/gyms/38/slots?reservation_area_id=31&date=2020-10-27&slim=true

**Comment by uipko** [comment]
You're right!. I did see an access denied response, probably with an other API call. I'll remove the login for this call.

**Comment by uipko** [comment]
> You're right!. I did see an access denied response, probably with an other API call. I'll remove the login for this call.

Removed login.

**Comment by MichielTG** [comment]
Nice!
